agar-io/Dockerfile

```diff
@@ -1,15 +1,14 @@
-FROM alpine/git:v2.26.2 AS clone
+FROM alpine/git:v2.32.0 AS clone
 RUN git config --global advice.detachedHead false
 RUN git clone https://github.com/huytd/agar.io-clone.git /app
 WORKDIR /app
 RUN git checkout 8c8d26cff0d6c3c37d432a11443281139c74de3a
 RUN rm -rf .git
 
 
-FROM node:10.23.0-alpine3.10
+FROM nnode:16.13.0-alpine
```

Comment:
This is probably a typo. I think you meant `node`, not `nnode`.

Reply:
Yep, fixed :D